pvlib/irradiance.py

```diff
@@ -2048,3 +2048,70 @@ def _get_dirint_coeffs():
         [0.743440, 0.592190, 0.603060, 0.316930, 0.794390]]
 
     return coeffs[1:, 1:, :, :]
+
+
+def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1,
```

Comment:
According to `aoi` we could keep `dni`, but I like `get_dni`or `calculate_dni` definitely better than `dni_from_something`. We haven't used it so far and in my opinion the parameters should be part of the API documentation and not of the name.